index.js

```diff
@@ -234,14 +236,15 @@ function SpotifyWebHelper(opts) {
 	};
 	this.player.seekTo = seconds => {
 		this.status.playing_position = seconds; // eslint-disable-line camelcase
+		this.player.emit('seek', seconds);
 		return this.player.play(this.status.track.track_resource.uri + '#' + parseTime(seconds));
 	};
 	this.status = null;
 	var seekingInterval = null;
 	var startSeekingInterval = function () {
 		seekingInterval = setInterval(() => {
-			this.status.playing_position += 0.25; // eslint-disable-line camelcase
-		}, 250);
+			this.status.playing_position += SEEK_INTERVAL_LENGTH / 1000; // eslint-disable-line camelcase
```

Comment:
Fixed it with https://github.com/onetune/spotify-web-helper/pull/28/commits/c4a44c2050dffbb0bd198b63bcde0e8b758a1c44 just didn't commit that line I guess. 😊